Utilities/Distribution/MacARM/build_all.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+Build_Root=$HOME
+SimpleITK_Source=$Build_Root/SimpleITK
+COREBINARYDIRECTORY=$Build_Root/SimpleITK-build
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
```

Comment:
Please support SIMPLEITK_GIT_TAG similar to what's done here: https://github.com/SimpleITK/SimpleITK/blob/4937f5e907da600a8986cbb21285e9906bfa717a/Utilities/Distribution/manylinux/run.sh#L10-L15

Also look at who SRC_DIR ( in this script SimpleITK_Source) is defined relative to the current script. When all done locally, I think a temp dir will be needed for the cloned repo.